src/test/scala/chiselTests/Vec.scala

```diff
@@ -517,4 +517,19 @@ class VecSpec extends ChiselPropSpec with Utils {
   property("reduceTree should preserve input/output type") {
     assertTesterPasses { new ReduceTreeTester() }
   }
+
+  property("Vecs of empty Bundles should work") {
+    class EmptyBundle extends Bundle
+    val chirrtl = ChiselStage.emitChirrtl(new Module {
+      val idx = IO(Input(UInt(2.W)))
+      val in = IO(Input(new EmptyBundle))
+      val out = IO(Output(new EmptyBundle))
+
+      val reg = RegInit(0.U.asTypeOf(Vec(4, new EmptyBundle)))
+      reg(idx) := in
+      out := reg(idx)
+    })
```

Comment:
I think that's a fair point. I actually included them on purpose to make sure it wouldn't error (because what good is supporting empty Vecs if they then just error as soon as you try to dynamically index them or connect to them?)

What do you think about this reasoning?

Reply:
I'm fine with that reasoning. 👍